homeassistant/components/ntfy/notify.py

```diff
@@ -0,0 +1,153 @@
+"""ntfy platform for notify component."""
+from http import HTTPStatus
+import logging
+from typing import Any
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    ATTR_DATA,
+    ATTR_MESSAGE,
+    ATTR_TITLE,
+    ATTR_TITLE_DEFAULT,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+from homeassistant.const import (
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_URL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+CONF_TOPIC = "topic"
+DEFAULT_URL = "https://ntfy.sh"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(
+            CONF_TOPIC
+        ): cv.string,  # have to set topic globally in the configuration, can override in "data"
```

Comment:
Small nit, but might make this more readable to have the comment on the preceding line so `black` won't wrap the code.